package.json

```diff
@@ -33,7 +33,8 @@
     "request": "~2.81.0",
     "semver": "~5.3.0",
     "svgo": "~0.7.1",
-    "xml2js": "~0.4.16"
+    "xml2js": "~0.4.16",
+    "jsonpath": "^0.2.12"
```

Comment:
Could you keep this list alphabetized? It keeps the diffs clean, since that's what `npm install --save` does.

Reply:
No problem, also not sure if it should be `"jsonpath": "^0.2.12"` or `"jsonpath": "~0.2.12"`?